devon4j-http-rest-client/index.asciidoc

```diff
@@ -114,59 +161,45 @@ After getting response, you can handle the response further in your implementati
 An application needs to be configurable in order to allow internal setup but also to allow externalized configuration of a deployed package (e.g. integration into runtime environment). 
 Now, You need to modify the content of existing properties files and add configuration for Server.
 [step]
+== Modify properties for server configuration
 --
 changeFile("httprestclient/core/src/main/resources/application.properties", { "file": "files/client_application.txt" })
 changeFile("httprestclient/core/src/main/resources/config/application.properties", { "file": "files/client_config_application.txt" })
 --
-
 ## Service Discovery
 **service.client.default.url** :- It is used to set the default url of server and it is added for service discovery.
+
 **service.client.app.httprestserver.url** :- This property provide base url of REST in your application. It follows format such as "service.client.app.«application».url". Here, «application» refers to the technical name of the application providing the service.
 
 ## Timeouts
 **service.client.default.timeout.connection** :- It is used to set the default timeout for particular connection.
+
 **service.client.default.timeout.response** :- It is used to set the default timeout for particular response.
 
 ## Headers
 **service.client.app.httprestserver.auth** :- It is used for customization of Service Header. Here it is used for basic authentication.
 
 ## Authentication
-**service.client.app.httprestserver.user.login** :- It is used to set username of server for authentication
+**service.client.app.httprestserver.user.login** :- It is used to set username of server for authentication.
+
 **service.client.app.httprestserver.user.password** :- It is used to set password.
-Now, after configuration you have to build and then start the client application.
 ====
 
-====
-You have successfully built the devon4j rest client.
-Let's build and start the service client application.
+
+Now, Let's build and start the service client application.
 This might take some time for application to start.
 [step]
-== Build and Start devon4j REST Client
+== Build devon4j CXF REST Client
 --
 buildJava("httprestclient", false)
 --
 
-* Once, application builds successfully. Open new terminal by clicking "+" and wait for it to load.
-* Now, Copy and execute below command to navigate into client server.
-`cd devonfw/workspaces/main/httprestclient/server`
-====
-
-====
-Now, Execute below command to start the application.
 
 [step]
 --
-runServerJava("httprestclient/server", { "startupTime": 1000, "port": 8081, "path": "httprestclient" })
+nextKatacodaStep("Start the CXF CLient Server", [{ "file": "files/startServer.md" }])

```

Comment:
Please do not use nextKatacodaStep. Everything you do with nextKatacodaStep is not tested. Everything you have done in this step can be done without nextKatacodaStep!